cmd/apiserver/app/server/run_server.go

```diff
@@ -42,7 +42,8 @@ func RunServer(opts *ServiceCatalogServerOptions) error {
 func runTPRServer(opts *ServiceCatalogServerOptions) error {
 	tprOpts := opts.TPROptions
 	glog.Infoln("Installing TPR types to the cluster")
-	if err := tpr.InstallTypes(tprOpts.clIface); err != nil {
+	tprInstaller := tpr.NewInstaller(tprOpts.clIface.Extensions().ThirdPartyResources())
```

Comment:
Sorry, missed this comment earlier.
By having it as an object (eventually as an interface{}), we are making it ready to be injected into other module constructors, it allows mocking when testing `run_server.go`(yet to be tested) or other modules that uses it.